styles/components/_layout.scss

```diff
@@ -411,3 +415,11 @@
 .posR {
   position: relative;
 }
+
+.borderRSm {
+  border-radius: 1px;
+}
+
+.borderRMd {
+  border-radius: 2px;
+}
```

Comment:
Let's also avoid border-radius classes, in most cases this should be taken care of by a container class.

In general, we're trying to avoid single-property classes, so we don't end up with hundreds of them.